src/types/rekor/index.ts

```diff
@@ -96,12 +94,11 @@ export const rekor = {
 
   toProposedHashedRekordEntry: (
     digest: Buffer,
-    signature: Buffer,
-    certificate: string
+    signature: SignatureMaterial
   ): HashedRekordKind => {
     const b64Digest = digest.toString('hex');
-    const b64Signature = signature.toString('base64');
-    const b64Certificate = enc.base64Encode(certificate);
+    const b64Signature = signature.signature.toString('base64');
+    const b64Key = enc.base64Encode(toPublicKey(signature));
```

Comment:
What format should the public key be in? Is the `publicKey.content` sent to Rekor a base64 encode DER key? In which case, should `signature.key.value` be a `Buffer` as DER should be binary?

Reply:
Just had a quick poke in node and looks like you get a Buffer if you export a `PublicKeyObject` with `publicKey.export({format: 'der', type: 'spki'})`

Reply:
Does this mess with `certificates` being strings?

Reply:
The `toPublicKey` function here simply extracts either the signing certificate or the key from the `SignatureMaterial` object. In both cases the values should be PEM strings which Rekor wants base64-encoded.